forecasting_sa/base_forecasting_conf.yaml

```diff
@@ -11,42 +11,48 @@ train_predict_ratio: 4
 freq: D
 
 tuning_enabled: false
-tuning_max_trials: 64
-tuning_parallelism: 8
+tuning_max_trials: 16
+tuning_parallelism: 4
 tuning_retrain: false
 tuning_distributed: true
-tuning_max_epochs: 50
-tuning_max_context_len: 120
+tuning_max_epochs: 10
+tuning_max_context_len: 40
 
-static_categoricals:
```

Comment:
One of the many changes in the GluonTS [api](https://ts.gluon.ai/stable/api/gluonts/gluonts.dataset.pandas.html?highlight=from_long_dataframe#gluonts.dataset.pandas.PandasDataset.from_long_dataframe)